pkg/api/api.go

```diff
@@ -578,6 +580,7 @@ func (hs *HTTPServer) registerRoutes() {
 		adminRoute.Get("/settings", authorize(ac.EvalPermission(ac.ActionSettingsRead)), routing.Wrap(hs.AdminGetSettings))
 		adminRoute.Get("/settings-verbose", authorize(ac.EvalPermission(ac.ActionSettingsRead)), routing.Wrap(hs.AdminGetVerboseSettings))
 		adminRoute.Get("/stats", authorize(ac.EvalPermission(ac.ActionServerStatsRead)), routing.Wrap(hs.AdminGetStats))
+		adminRoute.Get("/anonstats", authorize(ac.EvalPermission(ac.ActionServerStatsRead)), routing.Wrap(hs.GetAnonUsers))
```

Comment:
We should put api routes in the anonymous package as we do for our other services, and move the handlers there as well